common/client-libs/validator-client/src/nyxd/traits/vesting_signing_client.rs

```diff
@@ -184,6 +191,25 @@ impl<C: SigningCosmWasmClient + Sync + Send + Clone> VestingSigningClient for Ny
             .await
     }
 
+    async fn vesting_update_gateway_config(
+        &self,
+        new_config: GatewayConfigUpdate,
+        fee: Option<Fee>,
+    ) -> Result<ExecuteResult, NyxdError> {
+        let fee = fee.unwrap_or(Fee::Auto(Some(self.simulated_gas_multiplier)));
```

Comment:
`VestingSigningClient` is in this weird in-between state of refactoring lol. But you can remove the duplicate code by calling `execute_vesting_contract` (like we do in `vesting_update_mixnode_cost_params`). Then you don't have to worry about fees, etc.